zanata-war/src/test/java/org/zanata/ImmutableDbunitJpaTest.java

```diff
@@ -20,45 +20,42 @@
  */
 package org.zanata;
 
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.BeforeMethod;
+
+import org.junit.After;
+import org.junit.Before;
 
 /**
  * @author Sean Flanigan <a href="mailto:sflaniga@redhat.com">sflaniga@redhat.com</a>
  */
 public abstract class ImmutableDbunitJpaTest extends ZanataDbunitJpaTest {
 
-    @BeforeClass
+    @Before
```

Comment:
On 2015-06-02 13:56, Carlos Munoz wrote:
> In zanata-war/src/test/java/org/zanata/ImmutableDbunitJpaTest.java
> <https://github.com/zanata/zanata-server/pull/823#discussion_r31491550>:
> 
>>  
>>  /**
>>   * @author Sean Flanigan <a href="mailto:sflaniga@redhat.com">sflaniga@redhat.com</a>
>>   */
>>  public abstract class ImmutableDbunitJpaTest extends ZanataDbunitJpaTest {
>>  
>> -    @BeforeClass
>> +    @Before
> 
> Should this be JUnit's |@BeforeClass| instead?

That would be a more direct translation, but JUnit's BeforeClass can't
access non-statics.  This particular case shouldn't hurt test times too
much, because ImmutableDbunitJpaTest isn't used a lot.

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/823/files#r31491550>.
> 


-- 
Sean Flanigan

Principal Software Engineer
Localisation Tools Engineering
Red Hat